### PR TITLE
feat(eventing): host event specific details

### PR DESCRIPTION
**Pull request by datacore-vvarakantham**
Added host event specific details.

Dependency on PR: https://github.com/openebs/mayastor-dependencies/pull/54 (merged)

### CODE REVIEW

File: io-engine/src/eventing/host_events.rs [hunk 1]
```diff
@@ -15,10 +16,15 @@ use spdk_rs::NvmfController;
 
 impl EventMetaGen for NvmfSubsystem {
     fn meta(&self) -> EventMeta {
+        let nqn = self.get_nqn();
+        let (target, uuid) = match nexus_lookup_nqn(&nqn) {
+            Some(nexus) => ("nexus", nexus.uuid().to_string()),
+            None => ("replica", "".to_string()),
```
Comment (tiagolobocastro) [inline io-engine/src/eventing/host_events.rs]: why can't we get uuid for the replica?

Comment (datacore-vvarakantham) [reply io-engine/src/eventing/host_events.rs]: We can. I am adding it in the next commit.

Comment (datacore-vvarakantham) [reply io-engine/src/eventing/host_events.rs]: Added replica uuid now.

**Review by tiagolobocastro — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

**Comment by datacore-vvarakantham** [comment]
bors merge

**Pull request merged**